.alexignore

```diff
@@ -49,12 +49,12 @@ website/versioned_docs/version-0.43/
 website/versioned_docs/version-0.44/
 website/versioned_docs/version-0.45/
 website/versioned_docs/version-0.46/
-website/versioned_docs/version-0.47/
 website/versioned_docs/version-0.48/
 website/versioned_docs/version-0.49/
-website/versioned_docs/version-0.51/
+website/versioned_docs/version-0.50/
```

Comment:
Looks like you're adding back 50, 54, and 55 here accidentally, could you remove these or allow me to edit the branch?

Reply:
Also I already allowed edits from maintainers :)